test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -1172,6 +1172,19 @@ Describe "Extended FileSystem Item/Content Cmdlet Provider Tests" -Tags "Feature
             $result.Name | Should -BeExactly $testDir
         }
 
+        It "Verify Provider Root Directory + Force" {
```

Comment:
Please reword the title.
Really we fix and check edge scenario so it is better name it explicitly.